tests/SideBySide/CancelTests.cs

```diff
@@ -228,6 +228,18 @@ public void DapperQueryMultiple()
 			}
 		}
 
+		[Fact]
+		public async Task CancelCompletedCommand()
```

Comment:
What version of MySQL are you testing with? I tested this on 5.6.29. According to a [duplicate](https://bugs.mysql.com/bug.php?id=79838) of the bug I linked, the issue may be 

> repeatable with server version 5.1/5.5/5.6 and NOT REPEATABLE with server version 5.0 and 5.7